ansible_runner/runner_config.py

```diff
@@ -244,12 +264,29 @@ def prepare(self):
         self.env["RUNNER_OMIT_EVENTS"] = str(self.omit_event_data)
         self.env["RUNNER_ONLY_FAILED_EVENTS"] = str(self.only_failed_event_data)
 
+        if self.containerized:
+            debug('containerization enabled')
+            for container_setting in ('process_isolation_executable', 'container_image', 'container_volume_mounts'):
```

Comment:
What about `container_options`? shouldn't you debug those here too?

Reply:
+1 - container options should probably be debugged here, that and/or the full container invocation command. Maybe log `new_args` unless there's a concern of logging/leaking secret data.

Reply:
Good catch - updated